src/token.rs

```diff
@@ -73,20 +73,28 @@ impl<'a> Token<'a> {
         self.kind == TokenKind::StopWord
     }
 
-    /// Returns the number of graphemes in original token using number of characters in normalized
+    /// Returns the number of chars in original token using number of bytes in normalized
     /// token.
     ///
-    /// Grapheme clusters are counted in the pre-processed string (just before normalizing).
+    /// chars are counted in the pre-processed string (just before normalizing).
     /// For example, consider the string "Go💼od" which gets normalized to "gobriefcase od".
-    /// `num_graphemes_from_bytes(11)` for this token will return `3` - the number of characters in
+    /// `num_chars_from_bytes(11)` for this token will return `3` - the number of characters in
     /// the original string for 11 bytes in the normalized string.
     ///
     /// # Arguments
     ///
     /// * `num_bytes` - number of bytes in normalized token
-    pub fn num_graphemes_from_bytes(&self, mut num_bytes: usize) -> usize {
+    pub fn num_chars_from_bytes(&self, mut num_bytes: usize) -> usize {
         match &self.char_map {
-            None => self.word.len(),
+            None => {
+                // if we don't have a char_map, we look for the number of chars in the current
+                //   (probably normalized) string
```

Comment:
crap! yes! This is always normalized in our case 🤔; but, in another hand, if we don't deunicode, and so, we don't have char_map, there is no issue of counting normalized chars, mmmmmh... 🤔

Let's keep it like this :)

Reply:
Cool!